CHANGELOG.md

```diff
@@ -10,5 +10,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Let wmi_query and wmi_query_rsop return any error and do not consider it as error when a class is not found.
   [#26](https://github.com/greenbone/openvas-smb/pull/26)
 - Use results of pkgconfig for gnutls.[#30](https://github.com/greenbone/openvas-smb/pull/30)
+- Update INSTALL.md: Update reference system and add new dependency.[#32](https://github.com/greenbone/openvas-smb/pull/32)
```

Comment:
```suggestion
- Update INSTALL.md: Update reference system and add new dependency. [#32](https://github.com/greenbone/openvas-smb/pull/32)
```
Just noticed that this and the line above should have a space between `.` and `[`.